specs/Qowaiv.Specs/Date_specs.cs

```diff
@@ -120,6 +120,38 @@ public void to_WeekDate()
         => Converting.To<WeekDate>().From(Svo.Date).Should().Be(new WeekDate(2017, 23, 7));
 }
 
+public class Supports_JSON_serialization
+{
+    [TestCase("2012-04-23", "2012-04-23")]
+    [TestCase("2012-04-23", "2012-04-23T18:25:43.511Z")]
+    [TestCase("2012-04-23", "2012-04-23T10:25:43-05:00")]
+    public void System_Text_JSON_deserialization(Date svo, object json)
+        => JsonTester.Read_System_Text_JSON<Date>(json).Should().Be(svo);
+
+    [TestCase("2012-04-23", "2012-04-23")]
+    public void System_Text_JSON_serialization(object json, Date svo)
+        => JsonTester.Write_System_Text_JSON(svo).Should().Be(json);
+
+    [TestCase("2012-04-23", "2012-04-23")]
+    [TestCase("2012-04-23", "2012-04-23T18:25:43.511Z")]
+    [TestCase("2012-04-23", "2012-04-23T10:25:43-05:00")]
+    public void convention_based_deserialization(Date svo, object json)
+        => JsonTester.Read<Date>(json).Should().Be(svo);
+
+    [TestCase("2012-04-23", "2012-04-23")]
+    public void convention_based_serialization(object json, Date svo)
+        => JsonTester.Write(svo).Should().Be(json);
+
+    [TestCase("Invalid input", typeof(FormatException))]
```

Comment:
These tests use a different order of arguments (input, expected type of exception), compared to the order of arguments of previous tests (expectation, input).